### DISCUSSION TITLE
I2C Tutorial

### DISCUSSION THREAD

**Discussion by rawinza555** [nasa/fprime#810] [Show and tell]
I have recently made a tutorial on using I2C driver to interact with ADXL 345 accelerometer [here](https://github.com/rawinza555/fprime/blob/I2cTutorial/docs/Tutorials/I2cTutorial/Tutorial.md). I think this should covers the basic but let me know if I made any mistakes or something you think should be mentioned. It's somewhat still a work in progress but everything works on my end so I decided to share it in case anyone need extra resources.

**Comment by Joshua-Anderson** [comment]
This looks like a helpful example! @LeStarch what are your thoughts on adding an external tutorials section to the fprime documentation where we can link to tutorials like this?

**Reply by LeStarch** [reply]
Works for me.  I would label "Community Tutorials" and note that it is contributed by the community

**Reply by rawinza555** [reply]
Glad you guys like it!

**Comment by capsulecorplab** [comment]
Where was the tutorial move to? The original link redirects to a 404 page

**Reply by capsulecorplab** [reply]
@rawinza555 I can't seem to find your tutorial. Is it still on GitHub?

**Reply by ladiazusc** [reply]
I also cannot find this 🤔
